src/ResourceManager/Websites/Commands.Websites/Commands.Websites.csproj

```diff
@@ -215,6 +224,10 @@
       <Project>{3819d8a7-c62c-4c47-8ddd-0332d9ce1252}</Project>
       <Name>Commands.ResourceManager.Common</Name>
     </ProjectReference>
+    <ProjectReference Include="..\..\Profile\Commands.Profile\Commands.Profile.csproj">
```

Comment:
No, do not create a dependency on another cmdlets project in your cmdlets project